flytekit/core/python_auto_container.py

```diff
@@ -4,7 +4,7 @@
 import re
 from abc import ABC
 from dataclasses import dataclass
-from typing import Callable, Dict, List, Optional, TypeVar, Union
+from typing import Callable, Dict, List, Literal, Optional, TypeVar, Union
```

Comment:
Can we use `L` instead of `Literal`? i just know i'm going to get confused in the future